os/arch/arm/src/armv7-m/ram_vectors.h

```diff
@@ -61,6 +61,7 @@
 #include <arch/irq.h>
 
 #include "up_internal.h"
+#include "chip.h"
```

Comment:
Is this include required here? chip.h can be directly included in a file where its actually being used.